app/styles/helpers/_alignment.scss

```diff
@@ -1,3 +1,11 @@
+.bc-flex-container {
+  display: flex;
+}
+
+.bc-align-items-center {
```

Comment:
I know American is used elsewhere but my preference would be to use proper English in our naming. 😃

Reply:
We have decided to stick with American English for CSS following the standard set by `nhsuk-frontend` and `govuk-frontend`.